07-routing-gossip.md

```diff
@@ -178,7 +198,10 @@ valid signature using `node-id` of the double-SHA256 of the entire
 message following the `signature` field (including unknown fields
 following `alias`), and MUST NOT further process the message.
 
-The receiving node SHOULD ignore `ipv6` if `port` is zero.
+The receiving node SHOULD ignore the first `address descriptor` which
```

Comment:
So `addrlen` needs to be the size in bytes of the `address` array, and not the number of addresses, otherwise attaching new fields to the end would be broken. I first assumed it would be the latter case, maybe we can make this explicit above?